packages/edit-navigation/src/hooks/use-navigation-editor-menu.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useContext } from '@wordpress/element';
+/**
+ * Internal dependencies
+ */
+import { MenuIdContext, untitledMenu } from './index';
+
+export default function useNavigationEditorMenu() {
```

Comment:
I feel like `useNavigationEditorMenu` has a lot of overlap with `useMenuEntity` and `useNavigationEditor` (and the names are confusing).

Maybe this can be combined with `useMenuEntity`?

Reply:
I think it would be better to keep them separate, not to make a function (hooks) too complex. But it is a good point that naming can be confusing so I have renamed use-navigation-editor-menu to use-selected-menu-data.js